ui-backend/catalog-ui-search/src/main/resources/OSGI-INF/metatype/org.codice.ddf.catalog.ui.xml

```diff
@@ -370,50 +370,56 @@
                 default="DDF"/>
 
         <AD id="product"
-                name="Product Name"
-                description="Specifies a custom product name for the UI, by default Intrigue"
-                type="String"
-                default="Intrigue"/>
+            name="Product Name"
+            description="Specifies a custom product name for the UI, by default Intrigue"
+            type="String"
+            default="Intrigue"/>
 
         <AD
                 description="The Branding."
                 name="Landing Page Background"
                 id="landingPageBackgroundSrc"
                 type="String"
                 default=""
-                />
+        />
 
-         <AD
+        <AD
                 description="The Branding."
                 name="Top Left Logo"
                 id="topLeftLogoSrc"
                 type="String"
                 default=""
-                />
+        />
 
-                 <AD
+        <AD
                 description="The Branding."
                 name="Bottom Left Logo"
                 id="bottomLeftLogoSrc"
                 type="String"
                 default=""
-                />
+        />
 
-                 <AD
+        <AD
                 description="The Branding."
                 name="Bottom Left Background"
                 id="bottomLeftBackgroundSrc"
                 type="String"
                 default=""
-                />
+        />
 
-                <AD
+        <AD
                 description="The Branding."
                 name="Menu Icon"
                 id="menuIconSrc"
                 type="String"
                 default=""
-                />
+        />
+
+        <AD id="extra"
```

Comment:
❓ Curious on the reasoning for making this a generic extra rather than more specific to inputDependencyRulesJson?

Reply:
Currently, that configuration is only being used downstream, and I figured it would be convenient for us to have a generic configuration object that downstream projects can utilize without having to make changes to ddf-ui.